code/src/components/Form.js

```diff
@@ -0,0 +1,238 @@
+import React, { useState } from 'react';
+import InputSelect from './InputSelect';
+import InputText from './InputText';
+import InputRadio from './InputRadio';
+import InputCheckbox from './InputCheckbox';
+import InputRange from './InputRange';
+import InputEmail from './InputEmail';
+import Button from './Button';
+import Summary from './Summary';
+import ProgressBar from './ProgressBar';
+import '../Styles/Form.css';
+
+export const Form = () => {
+	const [question, setQuestion] = useState(-1);
+	const [name, setName] = useState('');
+	const [season, setSeason] = useState('');
+	const [months, setMonths] = useState([]);
+	const [value, setValue] = useState(0);
```

Comment:
maybe the naming of this state variable could be a bit more information then "value", i can't say by looking at it what it contains.

Reply:
Yes, it's not very descriptive

Reply:
Very true, thank you for that feedback, I have changed my naming of this state!